### ISSUE TITLE
Minor Fixes

### ISSUE THREAD

**Issue by SouravBiswal** [techrishikb/CarPrice_LinearRegression#1]
Hey! This project is awesome. I was just going through your code and found some minor errors/typos. It'd be great if you could let me do some tweaks. Also I'm participating in hacktoberfest so generating a pull request would help me a lot.

**Comment by techrishikb** [comment]
Kindly let me know the cahnges you are expecting to be done please..

**Comment by SouravBiswal** [comment]
Yeah, I'll generate a pull request, you could see the things I've changed. Okay? If you like the changes you can merge it with the main branch.

**Comment by techrishikb** [comment]
ok
